website/__tests__/redux/integration/ElementIntegrationTests.jsx

```diff
@@ -125,7 +135,9 @@ describe('PositionsIntegrationTests.', () => {
     const firebaseApp = initFirebase();
 
     // dispatch and persist the changes to firebase
-    testStore.dispatch(updatePositionAndPersist(elemId, updatedLoc)).then(() => {
+    testStore.dispatch(updateAndPersist(updatePosition, elemId, updatedLoc))
+        .then(() => {
+      /* TODO: Firebase should be mocked! Not a real call to Firebase! */
```

Comment:
for now though, make this test async as well and add a done() call after deleting firebase.